### PR TITLE
#3008 allow gateways to be activated/deactivated on chain

### LINKED ISSUE #3008: Allow gateways to be activated/deactivated on-chain
Two tasks:

1. Cut out the logic from NewsHeadline that loads the on-chain configuration from the asset description. Put it in `lib/chain/onChainConfig` and provide `get` functions, e.g. the notifications are made accesible by exporting `getNotifications` from `onChainConfig`
2. Add a `getGateways` to the `onChainConfig` which returns the key `gateways`. `gateways` is a dict, e.g. could look like
```
gateways = {
    someGatewayId: {
        enabled: false
    }
}
```
3. Gateway configuration in `gateways.js` contains `availableGateways` and in it for each gateway a `isEnabled` flag. On startup, and after connecting to a chain, check if the on chain config has an entry for each gateway, and if `getGateways().gatewayid.enabled == false` then set `isEnabled` to `false`.

### PR DESCRIPTION
**Pull request by sschiessl-bcp**
<h2>General</h2>
Closes #3008

 - Required getting familiar with the gateway system

https://github.com/blockchainprojects/bitshares-ui/pull/179

<h2>Bounty Request</h2>

Bounty request: 4.5
Hours spent: 10.5

### CODE REVIEW

**Comment by sschiessl-bcp** [comment]
Requesting review of @startailcoon or @dot5enko pretty please

**Comment by dot5enko** [comment]
hi @sschiessl-bcp, yeah, looks good to me. I've already pushed some changes into my repo which change how the updateGatewayBackers bahave but there is no reason for you to wait for me - i'll rebase my changes on top of what you done. Looks like you're found a solution to get a key value storage work into blockhain !)

File: app/components/Account/AccountDepositWithdraw.jsx [hunk 1]
```diff
@@ -16,7 +16,6 @@ import AccountStore from "stores/AccountStore";
 import SettingsStore from "stores/SettingsStore";
 import SettingsActions from "actions/SettingsActions";
 import {openledgerAPIs} from "api/apiConfig";
-import BitKapital from "../DepositWithdraw/BitKapital";
```
[the remark this answers is not in the document]
Comment (sschiessl-bcp) [reply app/components/Account/AccountDepositWithdraw.jsx]: Website is offline, assets I removed last release

File: app/components/Account/AccountDepositWithdraw.jsx [hunk 2]
```diff
@@ -453,6 +457,7 @@ class AccountDepositWithdraw extends React.Component {
 
         serList.push({
             name: "Citadel",
+            identifier: "CITADEL",
             template: (
                 <div>
                     <div className="content-block">
```
[the remark this answers is not in the document]
Comment (sschiessl-bcp) [reply app/components/Account/AccountDepositWithdraw.jsx]: Name could have spaces and whatnot that are not suitable for identifier, also Blocktrades is TRADE e.g.. So derivation is not possible.

[the remark this answers is not in the document]
Comment (dot5enko) [reply app/components/Account/AccountDepositWithdraw.jsx]: everything okay. i thought `template` is the same, but its not

**Pull request merged**